CMakeLists.txt

```diff
@@ -57,3 +57,8 @@ add_test(osmxTest osmxTest)
 install(TARGETS osmx DESTINATION bin)
 add_custom_target(archive COMMAND dist/archive.sh ${OSMX_VERSION} ${CMAKE_SYSTEM_NAME})
 add_dependencies(archive osmx)
+
+add_library(osmx-static STATIC src/storage.cpp src/expand.cpp src/extract.cpp src/update.cpp src/region.cpp)
```

Comment:
any reason we shouldn't just do `add_library(osmx STATIC...` so that the output is libosmx.a instead of libosmx-static.a ?